script.js

```diff
@@ -0,0 +1,73 @@
+// -----------------Storing information in Js object --------------------------------
+
+const projects = [
+  {
+    ctaId: 1,
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: ['./images/desktop/tonic-project-desktop.png', 'images/Snapshoot Portfolio.png'],
+    technologies: ['html', 'css', 'javascript'],
+    live: 'https://ridabensalem.github.io/',
+    source: 'https://github.com/ridabensalem/ridabensalem.github.io',
+  },
+  {
+    ctaId: 2,
+    name: 'Multi-Post Stories',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    image: ['./images/desktop/multi-post-stories-project.png', './images/prof-art-project.png'],
+    technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+    live: 'https://ridabensalem.github.io/',
+    source: 'https://github.com/ridabensalem/ridabensalem.github.io',
+  },
+  {
+    ctaId: 3,
+    name: 'Facebook 360',
+    description: 'Exploring the future of media in Facebook\'s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    image: ['./images/desktop/facebook-360-project.png', './images/pop-mobile.png'],
+    technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+    live: 'https://ridabensalem.github.io/',
+    source: 'https://github.com/ridabensalem/ridabensalem.github.io',
+  },
+  {
+    ctaId: 4,
+    name: 'Uber Navigation',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    image: ['./images/desktop/uber-navigation-project.png', './images/multi-post-project.png'],
+    technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+    live: 'https://ridabensalem.github.io/',
+    source: 'https://github.com/ridabensalem/ridabensalem.github.io',
+  },
+];
+//  -----------------Handiling the pop up functionality----------------------------
+const modalPopup = document.querySelector('.block-popup');
+const popupOpening = document.querySelector('.popup-open');
+const popupClosing = document.querySelector('.popup-close');
+function Display() {
+  modalPopup.style.display = 'block';
+}
+function closePopup() {
+  modalPopup.style.display = 'none';
+}
+popupOpening.addEventListener('click', Display);
+popupClosing.addEventListener('click', closePopup);
+
```

Comment:
- [OPTIONAL ] Kindly make sure the close popup icon is hiding the popup, as for now, it's only bringing on top of the page.
- [OPTIONAL ] Same for the for the see project buttons, only one is opening the popup
__Please note that the present changes are optional only because the previous reviewer didn't mention them, but they are crucial for this project__

Reply:
For the next reviewer please approve the project I have only one extra  review left, and I did a massive work rebuilding everything because the code was messy and giving unexcepted errors so still some small padding and margin to match Figma 100% but it matches it with 98%, thank you :smile: